packages/graphql/src/translate/connection-clause/create-sort-and-limit.ts

```diff
@@ -61,17 +61,20 @@ export function createSortAndLimitProjection({
         offset = undefined;
         firstArg = undefined;
     }
-    addSortAndLimitOptionsToClause({
-        optionsInput: { sort: [edgeSortFields], limit: firstArg, offset },
-        target: relationshipRef,
-        projectionClause: withStatement,
-    });
-
-    addSortAndLimitOptionsToClause({
-        optionsInput: { sort: [nodeSortFields] },
-        target: nodeRef,
-        projectionClause: withStatement,
+    nodeAndEdgeSortFields.forEach((sortField) => {
+        const [nodeOrEdge, sortKeyAndValue] = Object.entries(sortField)[0];
+        addSortAndLimitOptionsToClause({
+            optionsInput: { sort: [sortKeyAndValue], limit: firstArg, offset },
```

Comment:
:thinking: 
There is something weird here, we are passing the limit values, but after this loop we are using addSortAndLimitOptionsTOClause again with these values. Maybe we can remove them from here?

Reply:
So the two cases here are:
1. inside `nodeAndEdgeSortFields.forEach`: for queries of the format `actorsConnection(first: 5, sort: [{ node: { name: ASC } }])` where limit is combined with sorting args - excluding the limit values from here results in pagination not taking place
2. after `nodeAndEdgeSortFields `: when `limit` is present but sorting on edge/nodes is not - nodeAndEdgeSortFields will be an empty `[]` and the `.forEach` never takes place to add the `limit` in cypher

Reply:
Fair enough, thanks for the clarification